### DISCUSSION TITLE
How to set Reminder every other day?

### DISCUSSION THREAD

**Discussion by Edfgdz** [iSoron/uhabits#981] [Help & FAQ]
I have one habbit which needs me to do it every other day so i also want this app reminds me to do it every other day too.

But look like at present this app doesn't support reminder working every other day. 

So is there any way to do it?

**Comment by sfsworms** [comment]
So I check on my version, and the app doesn't send you a reminder if the activity is already completed. So if you set an habit to be done once every two days, and set a reminder, you will only get the reminder once every two days, as the habit will be counted a completed on the off day.
